recipe/builder.py

```diff
@@ -14,12 +14,23 @@ def install():
     """ populate the environment variable added by the patch and actually
         install with pip
     """
+
+    cuda_version = os.environ.get('cuda_compiler_version', None)
+
+    try:
+        float(cuda_version)
+        is_cuda = True
+    except (TypeError, ValueError):
+        is_cuda = False
+
     if WIN:
         cmake_args = (
             "-DDLIB_FORCE_MSVC_STATIC_RUNTIME=OFF\n"
             "-DDLIB_LINK_WITH_SQLITE3=OFF\n"
             "-DBUILD_SHARED_LIBS=ON\n"
             "-DDLIB_IN_PROJECT_BUILD=ON\n"
+            "-DCUDNN_INCLUDE_DIR=%LIBRARY_INC%\n"
+            "-DCUDNN_ROOT=%LIBRARY_PREFIX%\n"
         ).format(os.environ)
```

Comment:
As these are coming from `os.envrion`, think we might need this

Also typing the CMake variables sometimes helps with parsing issues

```suggestion
            "-DCUDNN_INCLUDE_DIR:PATH={LIBRARY_INC}\n"
            "-DCUDNN_ROOT:PATH={LIBRARY_PREFIX}\n"
        ).format(os.environ)
```